config/default.go

```diff
@@ -139,6 +139,8 @@ URI = "zkevm-prover:50061"
 
 [Executor]
 URI = "zkevm-prover:50071"
+MaxResourceExhaustedAttempts = 1
```

Comment:
The default value must be the value that we normally will use on testnet/mainnet. I see in the toml files that we are setting 3 attempts, therefore I understand that the default value must be also 3. Take in mind that one of the objectives is that the node, when no specifying a toml config file, must run with the default values for testnet/mainnet

Reply:
I am not sure about this case. I thing the default behaviour should be not retrying and so the value must be 1 by default, but I understand your point.

Reply:
In any case I think the default values (default.go) must match the config values used in the default public testnet/mainnet toml files that we provide. But we can set the value to 1 in both places or 3. I think to do the retry (3) is fine as default behaviour

Reply:
Updated default to 3.